here/README.md

```diff
@@ -60,6 +60,14 @@ We have HERE data going back to 2012, but there sure weren't as many people driv
 
 Though data are reported at 5-minute intervals, on any given link (aka road segment) there may not be a lot of observations. Longer time periods and roads with higher traffic volumes have more probe data, and therefore more accurate results, than short time periods on roads with light traffic. More probe data increases accuracy. There are some road segments in the City (cul-de-sacs and other lightly travelled roads) that have very few observations. Because of this, we calculate travel times for time periods that are at least three weeks or longer so that data from multiple days can be aggregated into a more accurate result.
 
+## Matching maps and data
+
+HERE updates their maps on an annual basis (usually) to keep up with Toronto's evolving street network. It's important to make sure that the traffic analytics data you're using matches the street network. This only applies when you're using the raw table `here.ta`.
+
+Refer to table `here.street_valid_range` for which street version correspond to what range of data in `here.ta`. This table updates annually along with the HERE map refresh.
```

Comment:
Where are the street data exactly? I'm just assuming this refers to `here_gis.streets_*` and `here_gis.streets_att_*`

Reply:
year it is just `here_gis.streets_*`

Reply:
Can you just make that explicit in the documentation?